scipy/stats/_distn_infrastructure.py

```diff
@@ -3855,3 +3873,437 @@ def get_distribution_names(namespace_pairs, rv_base_class):
+        self.objective_val = getattr(res, "fun", None)
+        if self.objective_val is None:
+            self.objective_val = nllf(res.x)
+        self.nllf = nllf
+
+    def __repr__(self):
+        keys = ["dist", "data", "nllf", "params", "objective_val",
+                "success", "message"]
+        m = max(map(len, keys)) + 1
+        return '\n'.join([key.rjust(m) + ': ' + repr(getattr(self, key))
+                          for key in keys if getattr(self, key) is not None])
+
+    def plot(self, ax=None):
+        """Visualize the fit result.
+
+        Superposes the PDF/PMF of the fitted distribution over a normalized
+        histogram of the data.
+
+        Available only if ``matplotlib`` is installed.
+
+        Parameters
+        ----------
+        ax : matplotlib.axes.Axes
+            Axes object to draw the plot onto, otherwise uses the current Axes.
+
+        Returns
+        -------
+        ax : matplotlib.axes.Axes
+            The matplotlib Axes object on which the plot was drawn.
+        """
+        try:
+            from matplotlib.ticker import MaxNLocator  # type: ignore[import]
+        except ModuleNotFoundError:
+            message = "matplotlib must be installed to use method `plot`."
+            raise ValueError(message)
+
+        if ax is None:
+            import matplotlib.pyplot as plt  # type: ignore[import]
+            ax = plt.gca()
+
+        fit_params = np.atleast_1d(self.params)
+        support = self.dist.support(*fit_params)
+        lb = support[0] if np.isfinite(support[0]) else min(self.data)
+        ub = support[1] if np.isfinite(support[1]) else max(self.data)
+
+        if self.discrete:
+            x = np.arange(lb, ub + 2)
+            y = self.pxf(x, *fit_params)
+            ax.vlines(x[:-1], 0, y[:-1], label='Fit Distribution PMF',
+                      color='C0')
+            options = dict(density=True, bins=x, align='left', color='C1')
+            ax.xaxis.set_major_locator(MaxNLocator(integer=True))
+
+        else:
+            x = np.linspace(lb, ub, 200)
+            y = self.pxf(x, *fit_params)
+            ax.plot(x, y, '--', label='Fit Distribution PDF', color='C0')
+            options = dict(density=True, bins=50, align='mid', color='C1')
+
+        ax.hist(self.data, label="Histogram of Data", **options)
+        ax.set_title(f"{self.dist.name} Fit")
+        ax.legend(*ax.get_legend_handles_labels())
+        return ax
+
+
+def fit(dist, data, shape_bounds=None, *, loc_bounds=None, scale_bounds=None,
+        optimizer=optimize.differential_evolution):
+    r"""Fit a discrete or continuous distribution to data
+
+    Given a distribution, data, and bounds on the parameters of the
+    distribution, return maximum likelihood estimates of the parameters.
+
+    Parameters
+    ----------
+    dist : scipy.stats.rv_continuous or scipy.stats.rv_discrete
+        The object representing the distribution to be fit to the data.
```

Comment:
Typo? You usually correct me so not sure 😅
```suggestion
        The object representing the distribution to be fitted to the data.
```

Reply:
I believe that the sentence is constructed such that the verb "fit" should be conjugated the same way as the past tense (even thought this isn't the past tense... couldn't figure out the name for it), and both "fit" and "fitted" are standard as past tense of fit. (Think of the phrases "to be done", "to be helped", "to be built", etc.)
Apparently, ["fit" is more common](https://grammarist.com/usage/fit-fitted/) in North America, which might explain why we would tend to write it differently.